nav-authsql/_run.detached.bat

```diff
@@ -1,7 +1,7 @@
 set /p host=<__privatereponame.txt
 set /p ImageName=<__imagename.txt
-set ContainerName=%ImageName:/=_%
+set ContainerName=%ImageName::=_%
```

Comment:
Yes, I can understand now. My mistake. This is a container name and the behavior is different. Thanks again!!!